ember/app/abilities/additional-email.js

```diff
@@ -0,0 +1,15 @@
+import BaseAbility from "mysagw/abilities/-base";
+
+export default class AdditionalEmailAbility extends BaseAbility {
+  get canAdd() {
+    return this.isStaff;
+  }
+
+  get canEdit() {
+    return this.isStaff;
+  }
+
+  get canDelete() {
+    return this.isStaff;
+  }
```

Comment:
Is there somehow a nicer way than returning `this.isStaff` everytime? Maybe you can use this:
```js
@readOnly("isStaff") canAdd;
```

Or something else?

Reply:
I kind of like the js only approach here but it also is a bit bloated with the getters IMO. But would also be ok to leave as is.

Reply:
I'll add the decorators.